tests/unit/iac/collection/test_iac_scan_collection.py

```diff
@@ -1,68 +1,136 @@
+import pytest
 from pygitguardian.iac_models import IaCFileResult, IaCScanResult, IaCVulnerability
 
 from ggshield.iac.collection.iac_diff_scan_collection import IaCDiffScanCollection
 from ggshield.iac.collection.iac_path_scan_collection import IaCPathScanCollection
-from ggshield.iac.collection.iac_scan_collection import CollectionType
+from ggshield.iac.collection.iac_scan_collection import (
+    CollectionType,
+    IaCScanCollection,
+)
 from ggshield.iac.iac_scan_models import IaCDiffScanEntities, IaCDiffScanResult
 
 
-def test_iac_path_scan_collection_type() -> None:
-    """
-    GIVEN an IaC path scan collection
-    WHEN accessing its type
-    THEN the type is CollectionType.PathScan
-    """
-    collection = IaCPathScanCollection("0", IaCScanResult("0", "path_scan", "", []))
-    assert collection.type == CollectionType.PathScan
+def _generate_empty_path_collection() -> IaCPathScanCollection:
+    return IaCPathScanCollection(
+        "3ac2985e-dcf9-49ff-92fb-943548268de8",
+        IaCScanResult("3ac2985e-dcf9-49ff-92fb-943548268de8", "path_scan", "1.8.0", []),
```

Comment:
nit: use named arguments (in general, we consider it necessary/usefull when we have 3 or more arguments). Especially when they have the same type (strings), it can be very easy to make mistakes in the order.